index.html

```diff
@@ -264,8 +264,8 @@
         </a>
       </div>
       <div class="col-md-6">
-        <p>ClojureBridge Berlin is organized with a lot of love by volunteers. We try our best to make the workshop a memorable event and bring more women* into programming.</p>
-        <p><i>ClojureBridge Berlin wird mit ganz viel Liebe von Freiwilligen organisiert. Wir versuchen unser Bestes, aus dem Workshop ein unvergessliches Event zu machen und mehr Menschen(FLTI) zum Programmieren anzuregen.</i></p>
+        <p>ClojureBridge Berlin is organized with a lot of love by volunteers. We try our best to make the workshop a memorable event and bring more people into programming.</p>
```

Comment:
Maybe something that expresses the diversifying character of the workshop?
`and bring a divers group of people into programming`